packages/material-ui-lab/src/ClockPicker/ClockNumber.tsx

```diff
@@ -61,20 +109,16 @@ const ClockNumber: React.FC<ClockNumberProps & WithStyles<typeof styles>> = (pro
   };
 
   return (
-    <span
-      className={clsx(classes.root, {
-        [classes.selected]: selected,
-        [classes.disabled]: disabled,
-        [classes.inner]: inner,
-      })}
+    <ClockNumberRoot
+      className={clsx(classes.root, className)}
       style={transformStyle}
+      styleProps={styleProps}
+      theme={theme as Theme}
```

Comment:
Why is the `theme` required here?

Reply:
@mnajdova because `theme` returning from `useThemeProps` has type of `ThemeWithProps` which is not compatible with `Theme` that styled component expect so there are 2 ways to fix this.

1. fix the `useThemeProps` to use `Theme` instead of `ThemeWithProps`
2. cast the theme as `Theme`

which one should we use?